demisto_sdk/commands/common/hook_validations/playbook.py

```diff
@@ -291,3 +298,57 @@ def is_using_instance(self) -> bool:
                     self.is_valid = False
                     return False
         return True
+
+    def is_script_id_valid(self, skip_dependencies=False):
+        """Checks whether a script id is valid (i.e id exists in set_id)
+        Args:
+            skip_dependencies (bool):  whether should skip id set validation or not
+        Return:
+            bool. if all scripts ids of this playbook are valid.
+        """
+        is_valid = True
+        id_set = self.get_id_set_file(skip_dependencies)
+        if not id_set:
+            print_warning("Playbook script id validation cannot run, could not read id_set.json")
+            is_valid = False
+            return is_valid
+
+        all_scripts_from_id_set = id_set.get("scripts")
+        all_tasks_from_pb = self.current_file.get('tasks', {})
```

Comment:
Removing words that add no further information
```suggestion
        pb_tasks = self.current_file.get('tasks', {})
```